spot-contracts/contracts/RolloverVault.sol

```diff
@@ -268,6 +274,10 @@ contract RolloverVault is
         if (!_rollover()) {
             revert InsufficientDeployment();
         }
+
+        // Cleanup rollover; In case that there remain excess seniors and juniors after a successful rollover,
+        // we recover back to underlying.
+        _redeemTranche(trancheIntoPerp);
```

Comment:
Is this likely to be significant? I would think tranches coming out of perp would be more important.

Reply:
Since we tranche the entire underlying balance using the deposit bond, we usually "over-tranche" and have excess As and Zs. 

This is usually `recovered` when the next `recoverAndRedeploy` is executed but folks who redeem will get some extra A/Zs which they have to meld manually.

Reply:
This does increase the gas cost of weekly ops though. Can go either way with this. Open to removing, since we have the option to `recoverAndRedeem` ..

Reply:
Ah got it. Let's keep it for now, but could see removing it depending on how the flows in v2 change wrt recovers.